processing/data_collection/gazette/spiders/base.py

```diff
@@ -26,8 +29,16 @@ def __init__(self, start_date=None, *args, **kwargs):
                     f"Unable to parse {start_date}. Use %Y-%m-d date format."
                 )
                 raise
-        else:
-            self.logger.info("Collecting all gazettes available")
+
+        if end_date is not None:
+            try:
+                self.end_date = datetime.strptime(end_date, "%Y-%m-%d").date()
+                self.logger.info(f"Collecting gazettes until {self.end_date}")
```

Comment:
Move this log to the end of the `__init__` method. Thus, it will always be printed. ;)

Reply:
Please, do the same if the log statement for the start_date